pkg/mountutil/mountutil_freebsd.go

```diff
@@ -66,3 +69,70 @@ func ProcessFlagTmpfs(s string) (*Processed, error) {
 func ProcessFlagMount(s string, volStore volumestore.VolumeStore) (*Processed, error) {
 	return nil, errdefs.ErrNotImplemented
 }
+
+// ProcessSplit splits volume mount information received through command line into respective source, destination
+// and optsRaw fields according to the volume type chosen and returns them accordingly.
+func ProcessSplit(s string, volStore volumestore.VolumeStore, res *Processed, src string, dst string, options []string) (string, string, []string, error) {
```

Comment:
This function is exactly the same for freebsd and linux.  Can we reduce this so the code isn't copy pasted in two places?

Reply:
Okay